wiki/People/Users_with_unique_titles/de.md

```diff
@@ -0,0 +1,365 @@
+
+## Mitwirkende
+
+Diese Nutzer erhielten ihre entsprechenden Titel dafür, dass sie einen wesentlichen Beitrag zu ihrem Fachgebiet geleistet haben oder in ihrem Fachgebiet eine beispielhafte Persönlichkeit sind.
+
+<!-- TODO: reasons -->
+
+| Nutzer | Titel |
+| :-- | :-- |
+| ![][flag_CA] [Azer](https://osu.ppy.sh/users/2155578) | World Cup Organiser |
+| ![][flag_CN] [Ballance](https://osu.ppy.sh/users/165946) | osu!painter <!-- achievements and some other graphics --> |
+| ![][flag_US] [Charles445](https://osu.ppy.sh/users/85000) | osu! Paragon |
+| ![][flag_US] [ChillierPear](https://osu.ppy.sh/users/9501251) | World Cup Organiser |
+| ![][flag_US] [Daru](https://osu.ppy.sh/users/32480) | osu!painter <!-- maria and yuzu and a bunch of artwork for old site --> |
+| ![][flag_GB] [JBHyperion](https://osu.ppy.sh/users/4879508) | osu!catch Paragon |
+| ![][flag_DE] [LastExceed](https://osu.ppy.sh/users/6232245) | Master of the Fade <!-- implemented fade in and hidden mod in osu!(lazer) and notable Fade In player --> |
+| ![][flag_BR] [LeoFLT](https://osu.ppy.sh/users/3668779) | World Cup Organiser |
+| ![][flag_US] [LuigiHann](https://osu.ppy.sh/users/1079) | Resident Skinner |
+| ![][flag_NO] [MillhioreF](https://osu.ppy.sh/users/941094) | Pro Tester |
+| ![][flag_US] [nuudles](https://osu.ppy.sh/users/21312) | iPhone Dev |
+| ![][flag_DE] [OnosakiHito](https://osu.ppy.sh/users/290128) | osu!taiko Paragon |
+| ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418) | osu! Paragon |
+| ![][flag_US] [RBRat3](https://osu.ppy.sh/users/307202) | osu!painter |
+| ![][flag_SG] [Shoegazer](https://osu.ppy.sh/users/2520707) | osu!mania Paragon |
+| ![][flag_NL] [statementreply](https://osu.ppy.sh/users/126198) | Pro Tester |
+
+## Andere
+
+Diese Nutzer bekamen ihre entsprechenden Titel aus verschiedenen Gründen.
+
+<!-- TODO: reasons -->
+
+| Nutzer | Titel |
+| :-- | :-- |
+| ![][flag_SH] [BanchoBot](https://osu.ppy.sh/users/3) | w00t p00t |
+| ![][flag_US] [Derekku](https://osu.ppy.sh/users/91341) | COOL DUDE <!-- old forum meme --> |
+| ![][flag_DE] [Loctav](https://osu.ppy.sh/users/71366) | Popcorn Fairy <!-- movie threater pictures in staff chats, or something --> |
+| ![][flag_AU] [phill_old](https://osu.ppy.sh/users/53) | ¿ |
+| ![][flag_US] [Toy](https://osu.ppy.sh/users/2757689) | Former osu! Champion: 2019 / Grilled Dodongo |
+| ![][flag_HM] [Zallius](https://osu.ppy.sh/users/55) | Boats |
+
+## Anmerkungen
+
+[^mao-note]: ![][flag_DE] [Mao](https://osu.ppy.sh/users/2204515) wurde mit dem Titel **Elite Mapper II** stattdessen ausgezeichnet, da er den den Titel "Elite Mapper" auf zwei verschiedene Arten erfolgreich erworben hat: Das Gewinnen des Wettbewerbs [A Labour of Love](/wiki/Contests/A_Labour_of_Love_Contest) und das Überschreiten der 6-Punkte-Schwelle im neuen [monatlichen Beatmapping-Wettbewerb-](/wiki/Contests/Monthly_Beatmapping_Contest)-System.
```

Comment:
```suggestion
[^mao-note]: ![][flag_DE] [Mao](https://osu.ppy.sh/users/2204515) wurde stattdessen mit dem Titel **Elite Mapper II** ausgezeichnet, da er den den Titel "Elite Mapper" auf zwei verschiedene Arten erfolgreich erworben hat: Das Gewinnen des Wettbewerbs [A Labour of Love](/wiki/Contests/A_Labour_of_Love_Contest) und das Überschreiten der 6-Punkte-Schwelle im neuen [monatlichen Beatmapping-Wettbewerb-](/wiki/Contests/Monthly_Beatmapping_Contest)-System.
```

Reply:
I think `Beatmapping-Wettbewerb-` at the end of the line should be `Beatmapping-Wettbewerbs`, otherwise there is one hyphen too many. I would appreciate it if you could adjust that in your suggestion.

Reply:
done.

Reply:
thanks!